docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -282,6 +284,13 @@ This single command will fire up all the services, including the API server and
 
 Once ready, open a web browser and **visit <http://localhost:8000>**. If the app loads, sign in. Congratulations – you're all set! You now have a copy of freeCodeCamp's entire learning platform running on your local machine.
 
+> [!NOTE]
+> If you are using gitpod you will have to run the following command on your terminal to get your gitpod localhost url.
+> ```console
+gp url 8000
+
+
+
```

Comment:
This is not necessary. The ports are automatically opened, and can be found in the VSCode UI, if wanted.

```suggestion
```

Reply:
Thanks for the feedback. I was just giving this suggestion because on my end I was unable to see it. I was able to see the local ports but not the gitpod url. But this might be similar to what you said in previous comments where if someone has issues they can refer to the provided documentation link